senders/telegram/send.go

```diff
@@ -37,9 +37,15 @@ func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.
 	sender.logger.Debugf("Calling telegram api with chat_id %s and message body %s", contact.Value, message)
 	chat, err := sender.getChat(contact.Value)
 	if err != nil {
+		if ok, errorBrokenContact := checkBrokenContactError(sender.logger, err); ok {
```

Comment:
It's pretty strange to see that we raise an error if `ok` is true. Probably we need to invert bool value in return of function and invert condition here?

Reply:
Additionally: do we really need this bool value? probably we can just return `SenderBrokenContactError` if contact is broken and nil otherwise. And this condition will become just 

```go
if errorBrokenContact := checkBrokenContactError(sender.logger, err); errorBrokenContact != nil {
```

Reply:
Or even better solution. Let's the function `checkBrokenContactError` will return `SenderBrokenContactError` if contact is broken or original error otherwise. And here we can remove this second if and use just

```go
if err != nil {
	return checkBrokenContactError(sender.logger, err)
}
```

Reply:
looks good)

Reply:
fixed